crates/e2e/tests/e2e/ethflow.rs

```diff
@@ -203,6 +210,57 @@ async fn eth_flow_tx(web3: Web3) {
     .await;
 }
 
+async fn eth_flow_tx_insufficient_fee(web3: Web3) {
+    let mut onchain = OnchainComponents::deploy(web3.clone()).await;
+
+    let [solver] = onchain.make_solvers(to_wei(2)).await;
+    let [trader] = onchain.make_accounts(to_wei(2)).await;
+
+    // Create token with Uniswap pool for price estimation
+    let [dai] = onchain
+        .deploy_tokens_with_weth_uni_v2_pools(to_wei(DAI_PER_ETH * 1_000), to_wei(1_000))
+        .await;
+
+    // Get a quote from the services
+    let buy_token = dai.address();
+    let receiver = H160([0x42; 20]);
+    let sell_amount = to_wei(1);
+    let intent = EthFlowTradeIntent {
+        sell_amount,
+        buy_token,
+        receiver,
+    };
+
+    let services = Services::new(onchain.contracts()).await;
+    services.start_protocol(solver).await;
+
+    let quote: OrderQuoteResponse = test_submit_quote(
+        &services,
+        &intent.to_quote_request(trader.account().address(), &onchain.contracts().weth),
+    )
+    .await;
+
+    let valid_to = chrono::offset::Utc::now().timestamp() as u32
+        + timestamp_of_current_block_in_seconds(&web3).await.unwrap()
+        + 3600;
+    let mut ethflow_order =
+        ExtendedEthFlowOrder::from_quote(&quote, valid_to).include_slippage_bps(300);
+    // set a fee amount that is lower than the quoted fee amount
+    ethflow_order.0.fee_amount = 1.into();
+
+    submit_order(&ethflow_order, trader.account(), onchain.contracts()).await;
+
+    let uid = ethflow_order.uid(onchain.contracts()).await;
+    let is_available = || async { services.get_order(&uid).await.is_ok() };
+    wait_for_condition(TIMEOUT, is_available).await.unwrap();
+
+    let order = services.get_order(&uid).await.unwrap();
+    assert_eq!(
+        order.metadata.onchain_order_data.unwrap().placement_error,
+        Some(OnchainOrderPlacementError::InsufficientFee)
+    );
```

Comment:
And those orders are filtered out from the auction? Maybe we should also assert that the uid doesn't show up in the current auction?

Reply:
Yes, there is a filter for orders with error. 
Added assert for empty auction.